AENS.md

```diff
@@ -125,7 +125,36 @@ This can have multiple entries, e.g. a payment address associated
 with the name and an oracle address associated with the name.
 
 
-### Name
+A subname entry (or, subdomain), belongs to a name entry.
+
+The fields **owner**, **expires_by** and **client_ttl** of the
+name entry applies to all subname entries of the name entry.
+
+The lifecycle of subnames are driven by the lifecycle of the name
+they belong to.
+
+Issuing `revoke` or `transfer` transactions on name causes revoking or
+transfering of all of the name's subnames.
+
+When the name enters `unclaimed` state, all of the name's subnames
+are deleted.
+
+Issuing `claim`, `revoke`, `transfer` and `update` transaction where
+subname is provided instead of name causes failure, and such transaction
+is not propagated to other nodes.
+
+The only subname specific fields are **pointers** which can be
+unique for every subname entry.
```

Comment:
I rather see a better specification here. Somehow they MUST be unique, because if you give mpultiple entries the implementation choses one of them. Also describing what a point is could be good.